Spanner/src/ValueMapper.php

```diff
@@ -62,6 +63,7 @@ class ValueMapper
         self::TYPE_JSON,
         self::TYPE_PG_NUMERIC,
         self::TYPE_PG_JSONB,
+        self::TYPE_FLOAT32,
```

Comment:
I'm not sure what's breaking here!

Reply:
I think the issue could be the place where you inserted this value. Could you try inserting this value to the end of the list rather than the middle?

Reply:
Now its end of list, but still the error persists.

Reply:
It seems this is wrongly flagged by roave bc compat checker. 
One approach @saranshdhingra suggested is to add an `@internal` flag on `ValueMapper::$allowedTypes` to suppress similar errors in future, yet to get a final approval for this.